hazelcast/src/main/java/com/hazelcast/internal/partition/PartitionRuntimeState.java

```diff
@@ -35,7 +35,7 @@
 
 public final class PartitionRuntimeState implements IdentifiedDataSerializable {
 
-    // used for writing state
+    /** Map from address to the replica index of the first partition where the address occurrs. Used for writing state. */
     private Map<Address, Integer> addressToIndexes;
 
     // used for reading state
```

Comment:
I tend to keep to this rule - if it is public, write a full multi-line javadoc. If it is private, write a javadoc but try to keep it single-line (not always applicable). WDYT? Should I avoid single-line javadoc and use comments for that? And keep multi-line javadoc on private methods if neccessary?